### ISSUE TITLE
14: Traceback on removing group by on editable list

### ISSUE THREAD

**Issue by sswapnesh** [odoo/odoo#70822]
Impacted versions: 14.0

Steps to reproduce:
Eg, Website --> Configuration --> Menu
![Peek 2021-05-15 22-35](https://user-images.githubusercontent.com/16624719/118372165-d025e680-b5cd-11eb-9b0b-6c2f1424984d.gif)

Current behavior:
```
Error:


Traceback:
TypeError: Cannot read property 'offsetParent' of null
    at http://localhost:8069/web/content/515-7acf3bd/web.assets_backend.js:2588:382
    at Array.forEach (<anonymous>)
    at Class._computeDefaultWidths (http://localhost:8069/web/content/515-7acf3bd/web.assets_backend.js:2588:320)
    at Class._freezeColumnWidths (http://localhost:8069/web/content/515-7acf3bd/web.assets_backend.js:2592:244)
    at Class.<anonymous> (http://localhost:8069/web/content/515-7acf3bd/web.assets_backend.js:2571:370)
    at Class.trigger (http://localhost:8069/web/content/496-38a0146/web.assets_common.js:4761:180)
    at Class.trigger (http://localhost:8069/web/content/496-38a0146/web.assets_common.js:4767:148)
    at http://localhost:8069/web/content/515-7acf3bd/web.assets_backend.js:2581:100
```
Expected behavior: No Traceback

Video/Screenshot link (optional):

Support ticket number submitted via odoo.com/help (optional):

**Comment by Yenthe666** [comment]
@ged-odoo

**Comment by msh-odoo** [comment]
@sswapnesh @Yenthe666 We already have a task for this issue and here is the PR: #70236

**Comment by Yenthe666** [comment]
Why 14.3 though?

**Comment by SimonGenin** [comment]
I just checked, and the bug appears from the 14.3. Everything works fine in 14.

**Comment by Yenthe666** [comment]
@sswapnesh you reported it for V14. Can you check if it is fixed there in the meanwhile or if you maybe tested on 14.3?

**Comment by SimonGenin** [comment]
Actually, I've just been able to reproduce. I will investigate, as it didn't happen in the test I did before in enterprise.

**Comment by SimonGenin** [comment]
So there is indeed two bugs: 
- one from 14.3 when trying to edit / add to the list and remove the group by
- one from 14 (maybe earlier, I'll check) when trying to edit a list with the sequence handle (or something related to the sequence) and remove the group by

**Comment by msh-odoo** [comment]
@SimonGenin @Yenthe666 We can close this as it should be fixed with #73219
